package_exporter/src/generate_package.rs

```diff
@@ -0,0 +1,195 @@
+use rmf_site_format::{Workcell, AssetSource, Geometry};
+use std::path::{PathBuf, Path};
+use dirs;
+use std::error::Error;
+use std::io::{Error as IoError, ErrorKind as IoErrorKind};
+use crate::template;
+
+pub fn generate_package(
+    workcell: &Workcell,
+    package_context: &template::PackageContext,
+    output_directory: &String,
+) -> Result<(), Box<dyn Error>> {
+
+    let output_directory = "output".to_string();
```

Comment:
Since you redefine `output_directory` here the argument is effectively ignored and the output folder is always `output` regardless of what the user specifies.

Reply:
Should be fixed in c822bd5.